Documentation/ApiOverview/SiteHandling/AddLanguages.rst

```diff
@@ -45,6 +45,18 @@ fallback levels - for example "Modern Chinese" might fall back to "Chinese (Trad
 Configuration Properties
 ========================
 
+enabled
+-----
```

Comment:
This headline would not be rendered properly, as the `-` is not as long as the headline.